src/components/learner-credit-management/AssignmentStatusTableCell.jsx

```diff
@@ -16,12 +17,16 @@ const AssignmentStatusTableCell = ({ row }) => {
     learnerState,
     errorReason,
   } = original;
-
   // Learner state is not available for this assignment, so don't display anything.
   if (!learnerState) {
     return null;
   }
 
+  // If learnerState is failed but no error reason is defined, don't display anything.
+  if (learnerState === 'failed' && !errorReason) {
+    return <FailedSystem />;
+  }
```

Comment:
Was not sure what to return here, whether a failed system chip or null.

Reply:
Falling back to failed system chip is reasonable IMO :)